src/strategies/layers/connector/morpho/MorphoConnector.sol

```diff
@@ -75,7 +75,12 @@ abstract contract MorphoConnector is ERC4626Connector {
   // slither-disable-next-line naming-convention,dead-code
   function _connector_init(address[] memory rewardTokens) internal {
     super._connector_init();
-    _rewardTokens = rewardTokens;
+    for (uint256 i = 0; i < rewardTokens.length; ++i) {
+      address rewardToken = rewardTokens[i];
+      _rewardTokens.push(rewardToken);
+      rewards[rewardToken] =
```

Comment:
We have to initialize tokens or they will be re-added to `_rewardTokens` when configured for the first time